exercises/practice/circular-buffer/circular-buffer.wat

```diff
@@ -1,12 +1,13 @@
 (module
-  (memory 1)
+  ;; Cap the memory at a single 64KiB page
+  (memory 1 1)
```

Comment:
Side note: is this (capping the memory) something we want to do for other exercises too?

Reply:
We can! It's mostly useful for preventing errors in loops that call `memory.grow`, so it's particularly important here. No harm in doing this more widely though. I'll do that as a follow-up PR.